src/containers/navigation-icons/user-icons/UserIcons.jsx

```diff
@@ -62,27 +62,35 @@ const UserIcons = ({ setSidebarOpen }) => {
   return (
     <Box ref={anchorRef} sx={styles.iconBox}>
       <Tooltip arrow title={t('iconsTooltip.language')}>
-        <IconButton disabled size='large' sx={styles.langIcon}>
-          <LanguageIcon color='disabled' />
-        </IconButton>
+        <Box>
+          <IconButton disabled size='large' sx={styles.langIcon}>
+            <LanguageIcon color='disabled' />
+          </IconButton>
+        </Box>
```

Comment:
Why are you wrapping each icon in Box component?

Reply:
It is redundant, maybe do a Box with disabled, cause in the future we can forget to delete it. And so we will definitely delete this block.